serverless/cmd/integrate/main.go

```diff
@@ -128,15 +127,24 @@ func main() {
 	}
 }
 
+func getKeyFile(path string) (key string, err error) {
```

Comment:
Don't typically name the return values unless there's a good reason (https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values)

Reply:
Done